ui/src/testing/stubs/user-api-stub.ts

```diff
@@ -0,0 +1,33 @@
+import {User, UserApi, UserResponse, UserRole} from 'generated/fetch';
+
+import * as fp from 'lodash/fp';
+
+export class UserApiStub extends UserApi {
+  existingUsers: UserRole[];
+  constructor(existingUsers?: UserRole[]) {
+    super(undefined, undefined, (..._: any[]) => { throw Error('cannot fetch in tests'); });
+    if (existingUsers) { this.existingUsers = existingUsers; }
```

Comment:
opt_nit: We don't have an official rule on this, but I would either do a one line without a brackets, or multiline with, but not a one-line with brackets.